totalRP3_Extended/script/script_operands.lua

```diff
@@ -376,6 +376,23 @@ local OPERANDS = {
 			["GetCameraZoom"] = "GetCameraZoom",
 		},
 	},
+	
+	["char_achievement"] = {
+		numeric = true,
```

Comment:
The operand shouldn't be numeric. Numeric operands allow the UI to use the <, >, <=, >= comparator which are not fit here.